ScanbotSDKExamples/NET/ClassicComponent.Droid/MainApplication.cs

```diff
@@ -0,0 +1,77 @@
+
+        // Use a custom temp storage directory for demo purposes.
+        public static TempImageStorageService TempImageStorage;
+
+        // TODO Add the Scanbot SDK license key here.
+        // Please note: The Scanbot SDK will run without a license key for one minute per session!
+        // After the trial period is over all Scanbot SDK functions as well as the UI components will stop working
+        // or may be terminated. You can get an unrestricted "no-strings-attached" 30 day trial license key for free.
+        // Please submit the trial license form (https://scanbot.io/sdk/trial.html) on our website by using
+        // the app identifier "io.scanbot.example.sdk.xamarin" of this example app.
+        const string LICENSE_KEY = null;
+
+        public MainApplication(IntPtr javaReference, JniHandleOwnership transfer) : base(javaReference, transfer)
+        { }
+
+        public override void OnCreate()
+        {
+            base.OnCreate();
+
+            TempImageStorage = new TempImageStorageService(GetExampleTempStorageDir());
+
+            Log.Debug(LOG_TAG, "Initializing Scanbot SDK...");
+            NativeDroid.ScanbotSDK.Initialize(this, LICENSE_KEY, new NativeDroid.Configurations.SBSDKConfiguration
+            {
+                EnableLogging = true,
+                Encryption = new SBSDKEncryption
+                {
+                    Mode = EncryptionMode.AES256,
+                    Password = "S0m3W3irDL0ngPa$$w0rdino!!!!"
+                }
+            });
+
+            ImageLoader.Instance = new ImageLoader(this);
+
+            // In this example we always cleanup the demo temp storage directory on app start.
+            TempImageStorage.CleanUp();
+        }
+
+        private string GetExampleTempStorageDir()
+        {
+            // !! Please note !!
+            // In this demo app we use the "ExternalStorageDirectory" which is a public(!) storage directory.
+            // All image files as well as export files (PDF, TIFF, etc) created by this demo app will be stored
+            // in a sub-folder of this storage directory and will be accessible for every(!) app having external storage permissions!
+            // We use the "ExternalStorageDirectory" here only for demo purposes, to be able to share generated PDF and TIFF files.
+            // (also see the example code for PDF and TIFF creation).
+            // If you need a secure storage for all images and export files (which is strongly recommended),
+            // use a suitable internal(!) storage directory.
+            //
+            // For more detais about the Android file system see:
+            // - https://developer.android.com/guide/topics/data/data-storage
+            // - https://docs.microsoft.com/en-us/xamarin/android/platform/files/
+
+            var external = GetExternalFilesDir(null).AbsolutePath;
+            var path = Path.Combine(external, "sbsdk-xamarin-cc-demo");
```

Comment:
@scan-mayank, does it make sense to keep the name as sbsdk-xamarin-cc-demo?

Reply:
Fixed